src/Bpi/ApiBundle/Resources/config/doctrine/ValueObject.Assets.Name.mongodb.yml

```diff
@@ -0,0 +1,5 @@
+Bpi\ApiBundle\Domain\ValueObject\Assets\Name:
+  type: embeddedDocument
+  fields:
+    Name:
```

Comment:
This should be `name` (all lowercase), right? The fields names in other `*.mongodb.yml` files are all lowercase.